scripts/check-submodule-version.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+SUBMODULES="cert clock healthcheck kvdb queue ticker"
```

Comment:
I wonder if there's a way to grab this using Go commands?

Reply:
Prob possible with some templating tricks using `go list -f`....I think this is good enough for now, a slim layer to help prevent issues like this in the future!

Reply:
Ah yes, good idea. Added some `go list` magic.